### ISSUE TITLE
Feature request: directly download in obsidian vault as obsidian web clipper

### ISSUE THREAD

**Issue by sql99** [cgranier/tabSidian#14]
Directly download in obsidian vault as obsidian web clipper, by choosing which folder to deposit the file in.

**Comment by cgranier** [comment]
Yes! This is coming in the next version. There's actually a number of features I may be borrowing from the clipper that should make tabSidian much more useful and easier to use.
